### PR TITLE
Migrate to go modules

### LINKED ISSUE #18: Migrate to modules
Getting this project onto modules would bring it in line with the future direction of Go.

### PR DESCRIPTION
**Pull request by guusvw**
Closes #18

* use go mod
* use go 1.13
* update kubectl
* updates vendors
* add .dockerignore

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,15 +1,19 @@
-FROM golang:1.11-alpine AS builder
+FROM golang:1.13-alpine AS builder
+
+WORKDIR /project
```
Comment (eytan-avisror) [inline Dockerfile]: what do we get from changing this?

File: .travis.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 language: go
 
 go:
-- 1.12.x
+  - 1.13.x
```
Comment (eytan-avisror) [inline .travis.yml]: This is a large change, and needs full testing of both node/pod reaper

File: Dockerfile (hunk 1 above)
Comment (guusvw) [reply Dockerfile]: Just working outside the `$GOPATH` to "proofe" that the switch to go modules work

**Comment by guusvw** [comment]
@eytan-avisror I'll setup a small test environment tomorrow and provide those setup.

**Comment by eytan-avisror** [comment]
@guusvw Awesome, thanks!

**Comment by eytan-avisror** [comment]
Hey @guusvw, were you able to test out your changes?

**Comment by eytan-avisror** [comment]
stale due to #19

**Pull request closed without merging**